### ISSUE TITLE
Make clear that this Library is Deprecated

### ISSUE THREAD

**Issue by DerGary** [xamarin/Xamarin.Auth#440]
It is not clear that this Library is not maintained any longer and can not be used when you want to upload your app to the apple store due to #423 #432 .
You can only know it if you find this comment: https://github.com/MicrosoftDocs/xamarin-docs/issues/2613#issuecomment-601227768

**Comment by Mephisztoe** [comment]
Yes. I only found out by accident when I tried to submit an app to the AppStore. In my case, I already moved to Xamarin.Essentials as hinted in #432.

However, my code still depends on the Account class provided by Xamarin.Auth which is why it is still referenced in my code.

**Comment by DerGary** [comment]
@Mephisztoe how did you upload it to the AppStore while you still depend on Xamarin.Auth? We currently encounter this Problem because we recently reworked our complete authentication using Xamarin.Auth and just found out that we can't upload the apps using this new authentication to the AppStore. So we need to rework our authentication again.

**Comment by friendly-tech** [comment]
Someone should fork this and continue maintaining it - Xamarin Essentials has nowhere near enough functionality (refresh tokens, PCKE)

**Comment by chrisfoulds** [comment]
I have apps that use this and still submit to the app store, why are you getting issues on submission ?

**Comment by friendly-tech** [comment]
I got this in my email from AppStoreConnect just today for a new app, i believe updates will start being rejected in december.

ITMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted. Instead, use WKWebView for improved security and reliability. Learn more (https://developer.apple.com/documentation/uikit/uiwebview)

**Comment by chrisfoulds** [comment]
oh that, either replicate it all and whip it out or wait till this gets merged.
https://github.com/xamarin/Xamarin.Auth/pull/443

**Comment by sujeetulli** [comment]
I also got the same message multiple times.

ITMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted. Instead, use WKWebView for improved security and reliability. Learn more (https://developer.apple.com/documentation/uikit/uiwebview)

Anyone is having any patch solution on this, Otherwise, I need to scrap my app to the bin. 

Help me guys, I trying to upload build in a test flight from last 1 week.

**Comment by friendly-tech** [comment]
For us, we moved to this approach
https://www.davidbritch.com/2020/04/authentication-from-xamarinforms-app_8.html?m=1
combined with 
https://devblogs.microsoft.com/xamarin/uiwebview-deprecation-xamarin-forms/

But we did have lots of issues with the linker, so adding plenty of `--linkskip=MyAssembly` to the mtouch args, also `--warn-on-type-ref=UIKit.UIWebView` is useful.

**Comment by chrisfoulds** [comment]
All I did was make a local copy, do the same changes as in that pull request to remove the old stuff (it already has the new stuff!) , Build and manually link the DLL. Took about 1hr.

**Comment by trinnguyen** [comment]
I sent that pull request. While waiting for it to be reviewed and merged, I use a DLL built locally to submit the new iOS version. Just give it a try as the workaround solution.

**Comment by chrisfoulds** [comment]
I submitted an update to Apple with my modified version of this library and I no longer get the warning so either they need to accept such a pull request or make it yourself
